app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,50 @@
+class Api::V1::CommentsController < ApplicationController
+  skip_before_action :verify_authenticity_token, only: [:create], if: -> { request.format.json? }
+
+  before_action :authenticate_user!
```

Comment:
- [ ] There is a little bug in your app. I successfully git the posts in json format but when I tried to hit the comments index method endpoint, it gives me `unauthorized error` even though I have signed up in the server and the app is running in local server. then I tried to comment these lines from 2 to 4 and I could successfully get the comments in json. kindly consider to fix that and try to hit the api endpoints to see if you are getting the json or not.
you can use either postman or thunder client extension in vscode.

Reply:
Hi! @Zuhaib042 I'm checking the code and I can see the comments in JSON format without the changes you suggested when a make the changes you are suggesting I'm not able to see the comments because I need the authentication, did you add the API token on the header? i the video I recorded you can see it!

Reply:
@Zuhaib042 [here is the link]( https://www.loom.com/share/b749129456df4541aa5a90423f34cf3c?sid=c7ab8fbb-8463-4718-aa3c-60974c91c0dc)

Reply:
@federicaulzurrun I apologize for the misunderstanding on my part. have a good day